zhmcclient/_lpar.py

```diff
@@ -427,6 +427,150 @@ def deactivate(self, wait_for_completion=True,
             self.wait_for_status(statuses, status_timeout)
         return result
 
+    @logged_api_call
+    def scsi_load(self, load_address, wwpn, lun, load_parameter=None,
+                  disk_partition_id=None,
+                  operating_system_specific_load_parameters=None,
+                  boot_record_logical_block_address=None,
+                  wait_for_completion=True, operation_timeout=None,
+                  status_timeout=None, allow_status_exceptions=False,
+                  force=False):
+        """
+        Load (boot) this LPAR from a designated SCSI device, using the
+        HMC operation "SCSI Load".
+
+        This HMC operation has deferred status behavior: If the asynchronous
+        job on the HMC is complete, it takes a few seconds until the LPAR
+        status has reached the desired value. If `wait_for_completion=True`,
+        this method repeatedly checks the status of the LPAR after the HMC
+        operation has completed, and waits until the status is in the desired
+        state "operating", or if `allow_status_exceptions` was
+        set additionally in the state "exceptions".
+
+        Authorization requirements:
+
+        * Object-access permission to the CPC containing this LPAR.
+        * Object-access permission to this LPAR.
+        * Task permission for the "SCSI Load" task.
+
+        Parameters:
+
+          load_address (:term:`string`):
+            Device number of the boot device.
+
+          wwpn (:term:`string`):
+            Worldwide port name (WWPN) of the target SCSI device to be
+            used for this operation, in hexadecimal.
+
+          lun (:term:`string`):
+            Hexadecimal logical unit number (LUN) to be used for the
+            SCSI Load.
+
+          load_parameter (:term:`string`):
+            Optional load control string.  If empty string or `None`,
+            it is not passed to the HMC.
+
+          disk_partition_id (:term:`integer`):
+             Optional disk-partition-id (also called the boot program
+             selector) to be used for the SCSI Load.
```

Comment:
Please state what the default is, if not specified. (The WS API book says the default is 0).

Reply:
Is the following description not better:  "If `None` it is not passed to the HMC."?

Because the defaults may change with newer HMC API versions.

Reply:
In addition, I've changed the condition from `if disk_partition_id:` to `if disk_partition_id is not None:` because `bool(0) == False`.